async-nats/src/lib.rs

```diff
@@ -264,7 +264,7 @@ pub use status::StatusCode;
 
 /// Information sent by the server back to this client
 /// during initial connection, and possibly again later.
-#[derive(Debug, Deserialize, Default, Clone, Eq, PartialEq)]
+#[derive(Debug, Deserialize, Serialize, Default, Clone, Eq, PartialEq)]
```

Comment:
ServerInfo is never serialized.

Reply:
In the server I'm writing, I'm sending it back to the client :).